File: 1887-reduction-operations-to-make-the-array-elements-equal/1887-reduction-operations-to-make-the-array-elements-equal.cpp
class Solution {
public:
    int reductionOperations(vector<int>& nums) {
        sort(nums.begin(), nums.end());
        int opr = 0;
        int i=1;
        int res =0;
        while(i<nums.size()){
            if(nums[i] == nums[i-1]){
                res += opr;
            }
            else{
                opr++;
                res += opr;
            }
            i++;
        }
        return res;
    }
};

File: 1887-reduction-operations-to-make-the-array-elements-equal/1887-reduction-operations-to-make-the-array-elements-equal.cs
public class Solution {
    public int ReductionOperations(int[] nums) {
        Array.Sort(nums);
        int opr = 0;
        int res =0;
        for(int i=1; i<nums.Length; i++){
            if(nums[i] == nums[i-1]) res += opr;
            else{
                opr++;
                res += opr;
            }
        }
        return res;
    }
}